nucypher/crypto/constants.py

```diff
@@ -21,6 +21,7 @@
 # Policy component sizes
 HRAC_LENGTH = 16
 SIGNATURE_SIZE = 64
+EIP712_MESSAGE_SIGNATURE_SIZE = 65
```

Comment:
I just realized we already have `LENGTH_ECDSA_SIGNATURE_WITH_RECOVERY = 65` in `blockchain/eth/constants.py`. Is it the same format, or just the same length?

Reply:
Arguably the same format, though `LENGTH_ECDSA_SIGNATURE_WITH_RECOVERY` is more ambiguous.

Reply:
Presumably it's plausible that we might later accept a different ECDSA signature with recovery, or even more than one of them.  Obviously there are many different bytes representations to go with different schemes.  Whereas, EIP712 is unambiguously a 65-byte standard.